src/lib/parser.js

```diff
@@ -231,7 +250,7 @@ export class Song {
           obj.chord = new Chord(cell);
           break;
         case '<':
-          cell = cell.substr(1, cell.length-2);
+          cell = cell.substring(1, cell.length-1);
```

Comment:
This looks like a functionality change. I don't remember why it was -2, but it would be good to prove that this is correct with a test.

Reply:
I know it's confusing but it's the same behavior; the second argument is "the end", before it was "the length".

- [String.prototype.substr (start, [, length])](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr)
- [String.prototype.substring (indexStart[, indexEnd])](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring)

```
'<foo>'.substr(1, '<foo>'.length-2);
"foo"
'<foo>'.substring(1, '<foo>'.length-1);
"foo" 
```

Reply:
Ooh thanks for clarifying. Confusing indeed!